### DISCUSSION TITLE
Telemon App- Better Monitoring, Better Health

### DISCUSSION THREAD

**Discussion by luppoalberto111** [StanfordBDHG/.github#56] [Show and Tell]
### Description

Hi!
We are thrilled to present you telemon - a universal solution for chronic disease management. We believe that better monitoring leads to better health conditions. To achieve this, we not only collect vital signs but also gather subjective feelings. For this purpose, we connect to a variety of Bluetooth-enabled medical devices to provide an even better user experience.

We were very excited when we discovered the Spezi project as it can help us speed up our development of new features.
Currently, we've included in our project `QuestionnairesOnFHIR` as we use it to collect valuable patient data.

![Image](https://github.com/StanfordBDHG/.github/assets/30599552/60228221-1d16-4ac5-9891-e9d6771e1e17)

Currently, we have in plan to include more parts of the Spezi project into our application so we are very thankful for the work of this team.

Also, we can't wait for upcoming innovations and ideas your project will bring.

### Contact Information and Additional Context

Everybody is more than welcome to download our solution and try it  [here](https://apps.apple.com/sk/app/telemon-chronic-care/id1641864064?l=sk). 

Every feedback is precious to us!

For more information feel free to check our [website](https://365care.io/en/) or our [social channel](https://www.facebook.com/telemon.care).

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
Very cool, thank you for sharing @luppoalberto111! 🚀

Great to hear that you are using `ResearchKitOnFHIR` and great to see it used in your app!
